src/opensles/AudioInputStreamOpenSLES.cpp

```diff
@@ -225,7 +226,7 @@ Result AudioInputStreamOpenSLES::close() {
     if (getState() == StreamState::Closed){
         result = Result::ErrorClosed;
     } else {
-        requestStop_l();
+        (void) requestStop_l();
```

Comment:
I haven't seem this idiom of casting to void before. I'm just curious about the reasoning behind this.

Reply:
It says "I know this function returns a result code. But I don't care because I will just keep going even if it returns an error code. So I am not even going to look at it.".